kill-stuck-pr-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -ex
+source ${CMS_BOT_DIR}/jenkins-artifacts
```

Comment:
source `_helper_functions.sh` too once the new functions are move to `_helper_functions.sh`